dj_hetmech_app/views.py

```diff
@@ -173,14 +173,27 @@ def get(self, request):
         metapath = request.query_params.get('metapath', None)
         if metapath is None:
             return Response(
-                {'error': 'metapth parameter not found in URL'},
+                {'error': 'metapath parameter not found in URL'},
                 status=status.HTTP_400_BAD_REQUEST
             )
+        try:
+            Metapath.objects.get(pk=metapath)
+        except:
+            return Response(
+                {'error': 'metapath abbreviation not found in database'},
+                status=status.HTTP_404_NOT_FOUND
+            )
```

Comment:
Can you undo this metapath check and revert to the TODO comment? The database does not contain all possible valid metapaths, so this is not the proper way to check.

Reply:
Sure.